_schema.graphql

```diff
@@ -4465,6 +4465,9 @@ type CommerceBuyOrder implements CommerceOrder {
   ): CommerceLineItemConnection
   mode: CommerceOrderModeEnum
   orderHistory: [CommerceOrderEventUnion!]!
+
+  # Whether Artsy collects taxes (e.g., sales tax or VAT) on this order.
```

Comment:
**chore**: I might be missing something, but I wonder why this comment is made for `paymentMethod` in schemas.

Reply:
ah, i'd ignore this because i manually edited the schema based on work @fladson was doing in exchange, planning to rebase after #4098 was merged.